vpr/src/route/rr_graph2.cpp

```diff
@@ -1203,53 +1203,53 @@ bool verify_rr_node_indices(const DeviceGrid& grid,
                     }
 
                     if (rr_graph.node_type(inode) == CHANX) {
-                        VTR_ASSERT_MSG(rr_node.ylow() == rr_node.yhigh(), "CHANX should be horizontal");
+                        VTR_ASSERT_MSG(rr_graph.node_ylow(rr_node.id()) == rr_graph.node_yhigh(rr_node.id()), "CHANX should be horizontal");
```

Comment:
Could use inode (and make it of type RRNodeId) to shorten the following lines, I think.  rr_node is just the rr_node associated with id inode, so this is a bit of a roundabout way to get back to inode.